submissions/chri911/popup-menu/style.css

```diff
@@ -0,0 +1,173 @@
+html {
+    box-sizing: border-box;
+}
+*,
+*::before,
+*::after{
+    box-sizing: inherit;
+}
+* {
+	margin: 0;
+	padding: 0;
+}
+a{
+    text-decoration: none;
+    
+}
+.container{
+    display: flex;
+    justify-content: flex-end;
+    padding: 10px 15px;
+    align-items: center;
+    color: #5f5c5f;
+    }
+.container > div{
+        margin-right: 20px;
+    }
+#popup,
```

Comment:
Avoid to style by `#id` or HTML tag. 
Try to separate responsibilities between HTML tag - markup, CSS class - style, and ID - JS logic or linking input to the label.